dapps/Cargo.toml

```diff
@@ -22,8 +22,10 @@ serde_json = "0.7.0"
 serde_macros = { version = "0.7.0", optional = true }
 zip = { version = "0.1", default-features = false }
 ethabi = "0.2.1"
+tiny-keccak= "1.0"
```

Comment:
Yes, it's a bit slower then `sha3` we use elsewhere, but it supports `update/finalize` workflow instead of loading everything to memory.